src/aiida_agents/tools/execution/introspection.py

```
@@ -20,10 +20,10 @@
 
 logger = logging.getLogger(__name__)
 
-__all__ = ["list_workflows", "describe_workflow"]
+__all__ = ["list_process_entry_points", "describe_process"]
```

Ruff (0.16.1)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)